plugins/DuskVerb/tools/tuner/combined_sweep.py

```
@@ -0,0 +1,349 @@
+                    x = x.astype(_np.float64) / _np.iinfo(x.dtype).max if x.dtype.kind == 'i' else x.astype(_np.float64)
+                    if x.ndim > 1:
+                        x = x.mean(axis=1)
+                    sos = _sg.butter(4, [800, 1200], 'band', fs=sr, output='sos')
+                    band = _sg.sosfiltfilt(sos, x)
+                    imp_band_db = 20 * _np.log10(max(_np.sqrt(_np.mean(band**2)), 1e-12))
+                    # Anchor impulse band ≈ -53 dB. Penalize deltas > +10 dB.
+                    overshoot = max(0.0, imp_band_db - (-43.0))
+                    loss += overshoot * 4.0
+                    measured["impulse_1k_band_db"] = imp_band_db
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)